lib/iris/fileformats/cf.py

```diff
@@ -74,6 +74,12 @@
                        ocean_s_coordinate_g1=['eta', 'depth'],
                        ocean_s_coordinate_g2=['eta', 'depth'])
 
+# Supported standard name modifiers and their associated units. Ref: [CF]
+# Appendix E.
+STD_NAME_MODIFIERS_UNITS = {'detection_minimum': 'u',
+                            'number_of_observations': '1',
+                            'standard_error': 'u',
+                            'status_flag': '1'}
```

Comment:
I'm unsure about this unit. The [StandardModifier table in the cf conventions](http://cfconventions.org/cf-conventions/cf-conventions.html#standard-name-modifiers) suggests there shouldn't be a unit. However, it also states that the `status_flag` modifier has been deprecated and should be replaced with a standard_name of status_flag which does have a unit of '1' (search for "status_flag" in the [cf-standard-name-table.xml](https://raw.githubusercontent.com/SciTools/iris/master/etc/cf-standard-name-table.xml))